src/components/userComponents/Exhibit/Exhibit.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Image from 'next/image';
+
+/**
+ *
+ * @param root0 passed in
+ * @param root0.title title of exhibit
+ * @param root0.description description of exhibit
+ * @param root0.image image
+ * @param root0.id id of exhibit
+ * @returns exhibit component
+ */
+export default function Exhibit({
+  title,
+  description,
+  image,
+  id,
+}: {
+  title: string;
+  description: string;
+  image: string;
+  id: number;
+}) {
+  return (
+    <li key={id} id={`a${id}`}>
+      <div className="w-[100%] px-4 py-8 bg-mint-cream rounded-lg flex-col justify-start items-start gap-2.5 inline-flex mt-6">
+        <div className="flex-col justify-start items-start gap-5 flex">
+          <div className="justify-start items-center gap-2 inline-flex">
+            <h2 className="text-night text-2xl font-semibold font-['Lato']">
```

Comment:
some of this styling isn't necessary.  See globals.css to see the default styling on h2 tags.  we do this to reduce redundant styling.